egov/egov-egf/src/main/java/org/egov/egf/utils/FinancialUtils.java

```diff
@@ -381,42 +382,32 @@ public AccountCodePurpose getAccountCodePurposeById(final Long id) {
     }
     
     @Transactional
-    public List<DocumentUpload> migrateUploadedFiles(RequestInfo requestInfo, List<DocumentUpload> docsUpload) {
+    public List<DocumentUpload> migrateUploadedFiles(RequestInfo requestInfo, List<DocumentUpload> docsUpload) throws FileNotFoundException {
         FileStoreMapper fileStoreS3 = null;
         FileStoreMapper fileStoreMapperNFS = null;
         final List<DocumentUpload> documentDetailsList = new ArrayList<>();
         for (DocumentUpload docs : docsUpload) {
-            try {
-                String fileStoreId = docs.getFileStore().getFileStoreId();
-                fileStoreMapperNFS = fileStoreMapperRepository.findByFileStoreId(fileStoreId);
-                // Here passing with the filestoreId and fetch file from the NFS
-                final File file = fileStoreService.fetchNFS(fileStoreMapperNFS.getFileStoreId(),
-                        FinancialConstants.FILESTORE_MODULECODE);
-                // Here taking the NFS file and pushing to S3 bucket
-                if (file.exists()) {
-                    fileStoreS3 = fileStoreService.store(file, fileStoreMapperNFS.getFileName(),
-                            fileStoreMapperNFS.getContentType(), FinancialConstants.FILESTORE_MODULECODE);
-                    LOGGER.info(("NFSFile-------------------" + fileStoreMapperNFS.getFileStoreId()));
-
-                    LOGGER.info("S3File-------------------" + fileStoreS3.getFileStoreId());
-                    docs.getFileStore().setFileStoreId(fileStoreS3.getFileStoreId());
-                    if (fileStoreS3 != null) {
-                        docs.setIsMigrated(true);
-                    }
-
-                    documentUploadRepository.save(docs);
-                }
-            } catch (Exception e) {
-                LOGGER.error("Exception while after pushing to S3 bucket filstoreId's : ", e);
+            String fileStoreId = docs.getFileStore().getFileStoreId();
```

Comment:
Keep this logic in try catch block and handle the exception by just printing the message to the log (use logger info) instead of throwing this to the caller method. If you throw the exception here, how it will continue with other records and what about the others those were already migrated in this transaction.

The flow should continue  even if one record fails in b/w. It should not stop executing other records. We can run again for the  failed ones as we are maintaining the migrated flag.